src/buttons/Forward/index.jsx

```diff
@@ -1,26 +1,45 @@
-import { Component } from 'preact'
-
+import PropTypes from 'prop-types'
 import Button from '../Button'
 import { ForwardIcon } from '@lana/b2c-mapp-ui-assets'
 import CSS from './styles.css'
 
-export default class Forward extends Component {
-	render() {
-		const dataTestId = this.props.dataTestId || null
-		return (
-			<section className={`${CSS.wrapper} ${this.props.className || ''}`}>
-				<Button
-					dataTestId={dataTestId}
-					id={this.props.id}
-					href={this.props.href}
-					onClick={this.props.onClick}
-					type={this.props.type}
-					loading={this.props.loading}
-				>
-					<ForwardIcon color="black-100" />
-					{this.props.children}
-				</Button>
-			</section>
-		)
-	}
+const Forward = ({ children, className, dataTestId, href, id, loading, onClick, type }) => {
```

Comment:
(naming) please consider renaming this to: `ForwardButton` for improved clarity and consistency